controllers/city.controller.go

```diff
@@ -9,25 +9,26 @@ import (
 	"project_weather/repositories"
 )
 
-type CityDB interface {
-	FindCityByID(id string) (*model.City, error)
-	RegisterCity(city *model.City) error
-	UpdateCity(city *model.City) error
-	DeleteCityByID(id string) error
-	GetAllCity() ([]model.City, error)
+type CityDBController interface {
```

Comment:
Name should be just `CityController` Now sound like AirplaneWingWheel :smile:

Reply:
Don't bother with `cityController`. Both `CityController` and `cityController` can live happy together